cypress/pages/table.page.js

```diff
@@ -0,0 +1,48 @@
+///<reference types = 'cypress'/>
+
+import { el } from "@faker-js/faker";
```

Comment:
is "el" short for element? try to name things with a meaning. 
faker is not used on this page, so this import should be removed